lib/nx_crc.c

```diff
@@ -466,7 +466,7 @@ uLong nx_crc32_combine64(crc1, crc2, len2)
 unsigned long crc32(crc, buf, len)
     unsigned long crc;
     const unsigned char FAR *buf;
-    uint64_t len;
+    uInt len;
```

Comment:
This is wrong in a 64-bit architecture because `uInt` becomes an `unsigned int`, which is 32-bit.

Reply:
@tuliom but `unsigned int` is exactly what is used by zlib. If the user needs something larger than 32 bits, it should be using `crc32_z` instead.

Reply:
Ack.  I'm resolving this conversation.
Thanks!